README.md

```diff
@@ -1,2 +1,144 @@
-# ada_spark_workflow
+# Ada SPARK Workflow
 A demo of best practices for open-source Ada/SPARK development
+
+[![codecov](https://codecov.io/gh/alire-project/ada_spark_workflow/branch/main/graph/badge.svg?token=9PZQ67LBPF)](https://codecov.io/gh/alire-project/ada_spark_workflow)
+
+The goal of this repository is to show the best practices for Ada/SPARK open-source development. We will do our best to keep it updated with the latest technologies and services from Alire and GitHub. Don’t hesitate to suggest enhancements in case we missed something interesting.
+
+In this project we use:
+
+ - [`Alire`](https://alire.ada.dev) for dependency management and release publication
+ - [`SPARK`](https://learn.adacore.com/courses/intro-to-spark/index.html) and [`GNATprove`](https://github.com/AdaCore/spark2014) for formal verification
+ - [`AUnit`](https://github.com/AdaCore/aunit) for unit-testing
+ - [`GNATcoverage`](https://github.com/AdaCore/gnatcoverage) for source code coverage analysis
+ - [`GitHub Actions`](https://docs.github.com/en/actions) for Continuous Integration
+
+
+# Layout of the repository
+
+This section provides a quick overview of the different parts of the projects.
+
+## Root directory
+The root directory contains the main library:
+
+ - `alire.toml` manifest file
+ - `ada_spark_workflow.gpr` GPRbuild project file (the name matches the Alire crate name)
+ - `src` directory that contains the source code for the library
+ - `README.md` file in markdown format displayed on the GitHub front page
+
+## `tests` directory
+
+This sub-directory contains the test-suite for the library. It is an Alire project itself, with a local dependency to the main crate in the root directory, as well as other dependencies that are only required for testing (e.g. `aunit` and `gnatcov`). In this directory we also find an `alire.toml` manifest, a GPRbuild project file and a source directory.
+
+## `.github` directory
+
+The `.github` directory contains files specific to GitHub features and services. For this project we use GitHub Actions for the Continuous Integration (see GitHub Actions section). But there are more to explore, like templates for opening issues and pull-requests, contributions or code-of-conduct guidelines.
```

Comment:
There is no "GitHub Actions" section. Also, can you use a link to the section?

Reply:
Added a link :+1: